CHANGELOG.md

```diff
@@ -14,6 +14,7 @@
 
 Breaking changes
 
+* @iov/base-types: Remove `TxId` in favour of `TxHash` in @iov/tendermint-rpc.
```

Comment:
@iov/tendermint-rpc cannot access BCP types. This is because @iov/tendermint-rpc should be useable for all kind of Tendermint-based applications. Thus we need a type for Tendermint transaction hashes. In tendermint "tx" and "hash" is the common naming. So we have the binary `TxHash` as a Tendermint-specific low level type that can then be hex-encoded for BCP type `TransactionId`.